Data Generator.py

```diff
@@ -0,0 +1,38 @@
+import random
+
+first_names = ["James", "Mary", "Liam", "Olivia", "Robert", "Patricia", "Noah", "Emma", "John", "Jennifer"]
+middle_names = ["Louise", "Rose", "Grace", "William", "Jane", "Elizabeth", "Thomas", "Anne", "Ann", "David"]
+last_names = ["Smith", "Johnson", "Williams", "Brown", "Jones", "Garcia", "Miller", "Davis", "Rodriguez", "Martinez"]
+
+names = []
+
+for i in range(10):
+    for j in range(10):
+        names.append(first_names[i] + ' ' + last_names[i])
+
+for i in range(10):
+    for j in range(10):
+        for k in range(10):
+            names.append(first_names[i] + ' ' + middle_names[j] + ' ' + last_names[k])
+
+countries = ["Switzerland", "Canada", "Sweden", "Australia", "United States", "Japan", "Germany", "New Zealand"]
+
+vaccines = [["Pfizer", "Mainz, Germany"], ["Moderna", "Massachusetts, U.S."], ["Novavax", "Maryland, U.S."]]
+
+random.shuffle(names)
+
+with open("E:/Vaccination DataBase/data_entry.tql", 'w') as f:
+
+    for i in range(3):
+        query = 'insert\n\t$v isa manufacturer,\n\thas vaccine_name "' + vaccines[i][0] + '",\n\thas factory_location "' + vaccines[i][1] + '";\n\n'

```

Comment:
Again, please use an array iterator rather than directly indexing:
```suggestion
    for vaccine in vaccines:
        query = 'insert\n\t$v isa manufacturer,\n\thas vaccine_name "' + vaccine[0] + '",\n\thas factory_location "' + vaccine[1] + '";\n\n'
```